src/engine/source/proto/include/eMessages/eMessage.h

```diff
@@ -5,13 +5,71 @@
 #include <variant>
 
 #include <base/error.hpp>
+#include <base/json.hpp>
 #include <google/protobuf/message.h>
 #include <google/protobuf/struct.pb.h>
 #include <google/protobuf/stubs/common.h>
 #include <google/protobuf/util/json_util.h>
 
 namespace eMessage
 {
+namespace detail
+{
+inline rapidjson::Value toRapidValue(const google::protobuf::Value& v, rapidjson::Document::AllocatorType& alloc);
```

Comment:
What's the point of these functions? Furthermore, it's exposing RapidJSON types, breaking the interface.